pkg/authorization/local_authorizer.go

```diff
@@ -41,6 +42,7 @@ type LocalAuthorizer struct {
 
 func NewLocalAuthorizer(versionedInformers kcpkubernetesinformers.SharedInformerFactory) (authorizer.Authorizer, authorizer.RuleResolver) {
 	a := &LocalAuthorizer{
+		// listers are saved in the struct here to ensure that informers are started and we do not encounter race conditions with them.
```

Comment:
nit: are instantiated early and ... with starting them

Reply:
Much clearer, thanks. Updated the comments.